test/config-template.test.js

```diff
@@ -0,0 +1,353 @@
+var should = require('chai').should();
```

Comment:
![oh stop it, you](https://cloud.githubusercontent.com/assets/30254/19976435/69a6b186-a1ce-11e6-9bd8-e0959c6c888c.png)